orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/DependsOnExecutionStage.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.pipeline;
+
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import com.netflix.spinnaker.orca.pipeline.tasks.DependsOnExecutionTask;
+import javax.annotation.Nonnull;
+
+public class DependsOnExecutionStage implements StageDefinitionBuilder {
+
+  public static String STAGE_TYPE = "dependsOnExecution";
```

Comment:
are we doing `STAGE_TYPE` or `PIPELINE_CONFIG_TYPE`?

Either way, probably needs to be `final`

Reply:
@marchello2000 which one should be used? I see both all over.